src/menu.cpp

```diff
@@ -742,9 +742,9 @@ KFighter player2fighter(int who)
      * it has no other power to begin with (the "weapon_elemental_effect" property
      * is 1-based: value of 0 means "no imbue".
```

Comment:
This comment doesn't match the code any more

Reply:
Fixed; thanks for spotting it!